composer.json

```diff
@@ -23,7 +23,7 @@
         "nikic/php-parser": "~3.0",
         "openeuropa/code-review": "^1.0.0-alpha4",
         "openeuropa/behat-transformation-context" : "~0.1",
-        "openeuropa/drupal-core-require-dev": "^8.6",
+        "openeuropa/drupal-core-require-dev": "~8.6@dev",
```

Comment:
Shouldn't this follow what's specified in `require`? `"drupal/core": "^8.6",`

Reply:
Corrected!